types/office-js-preview/index.d.ts

```diff
@@ -91274,51 +91584,57 @@ declare namespace PowerPoint {
         readonly items: PowerPoint.Slide[];
         /**
-         * Gets the number of slides in the collection.
+         * Adds a new slide at the end of the collection.
          *
          * [Api set: PowerPointApi BETA (PREVIEW ONLY)]
          * @beta
+         *
+         * @param options The options that define the theme of the new slide.
+         */
+        add(options?: PowerPoint.AddSlideOptions): void;
+        /**
+         * Gets the number of slides in the collection.
+         *
+         * [Api set: PowerPointApi 1.2]
          * @returns The number of slides in the collection.
          */
         getCount(): OfficeExtension.ClientResult<number>;
         /**
-         * Gets a slide using its unique ID. An exception is thrown if the slide does not exist.
+         * Gets a slide using its unique ID.
          *
-         * [Api set: PowerPointApi BETA (PREVIEW ONLY)]
-         * @beta
+         * [Api set: PowerPointApi 1.2]
          *
          * @param key The ID of the slide.
-         * @returns The slide with the unique ID. If such a slide does not exist, an exception is thrown.
+         * @returns The slide with the unique ID. If such a slide does not exist, an error is thrown.
          */
         getItem(key: string): PowerPoint.Slide;
         /**
          * Gets a slide using its zero-based index in the collection. Slides are stored in the same order as they
-                    are shown in the presentation. An exception is thrown if index is out of range.
+                    are shown in the presentation.
          *
-         * [Api set: PowerPointApi BETA (PREVIEW ONLY)]
-         * @beta
+         * [Api set: PowerPointApi 1.2]
          *
          * @param index The index of the slide in the collection.
-         * @returns The slide at the given index. An exception is thrown if index is out of range.
+         * @returns The slide at the given index. An error is thrown if index is out of range.
          */
         getItemAt(index: number): PowerPoint.Slide;
         /**
-         * Gets a slide using its unique ID. Returns an object whose `isNullObject` property is set to `true` if the slide does not exist.
+         * Gets a slide using its unique ID. If such a slide does not exist, an object with an `isNullObject` property set to true is returned. For further information,
+                    see {@link https://docs.microsoft.com/office/dev/add-ins/develop/application-specific-api-model#ornullobject-methods-and-properties | *OrNullObject methods
+                    and properties}.
          *
-         * [Api set: PowerPointApi BETA (PREVIEW ONLY)]
-         * @beta
+         * [Api set: PowerPointApi 1.2]
          *
          * @param id The ID of the slide.
-         * @returns The slide with the unique ID. If such a slide does not exist, an object whose `isNullObject` property is set to `true` is returned.
+         * @returns The slide with the unique ID.
```

Comment:
This change deletes a sentence that should stay because this is an "OrNullObject" property.